site/src/documents/pages/Page.tsx

```diff
@@ -33,7 +34,12 @@ export function Page(props: PropsWithLayout<GQLPageQuery>): JSX.Element {
                     canonicalUrl={`${process.env.NEXT_PUBLIC_SITE_URL}${props.pageContent?.path}`}
                 />
             )}
-            {document && document.__typename === "Page" ? <main>{<PageContentBlock data={document.content} />}</main> : null}
+            {document && document.__typename === "Page" ? (
+                <main>
+                    {document.stage && <StageBlock data={document.stage} />}
```

Comment:
Please only use `&&` with `boolean` values to prevent a falsy value from being rendered. 

```suggestion
                    {Boolean(document.stage) && <StageBlock data={document.stage} />}
```